### DISCUSSION TITLE
O大，求助，我用的三陶教育s30盒子刷armbian mcu是s905d2，刷s905X固件，内存识别788M，不准确。

### DISCUSSION THREAD

**Discussion by aylwin202406** [ophub/amlogic-s9xxx-armbian#2363] [Amlogic]
O大，求助，我用的三陶教育s30盒子刷armbian mcu是s905d2，刷s905X固件，内存识别788M，不准确。实际盒子是2+16，其他功能正常。请问我该如何做？

**Comment by ophub** [comment]
调整dtb的内存识别方法：
https://github.com/ophub/amlogic-s9xxx-openwrt/issues/602

u-boot添加方法看12.12:
https://github.com/ophub/amlogic-s9xxx-armbian/blob/main/documents/README.cn.md


参考上面的方法，添加u-boot或者调整dtb

**Comment by aylwin202406** [comment]
非常感谢。
________________________________
发件人: ophub ***@***.***>
发送时间: 2024年6月11日 11:30
收件人: ophub/amlogic-s9xxx-armbian ***@***.***>
抄送: aylwin202406 ***@***.***>; Author ***@***.***>
主题: Re: [ophub/amlogic-s9xxx-armbian] O大，求助，我用的三陶教育s30盒子刷armbian mcu是s905d2，刷s905X固件，内存识别788M，不准确。 (Discussion #2363)


ophub/amlogic-s9xxx-openwrt#602<https://github.com/ophub/amlogic-s9xxx-openwrt/issues/602>

参考上面的方法，添加u-boot或者调整dtb

―

**Reply by Hx0531** [reply]
搞好了吗，我也有个教育盒子 s905x的2+16.但是我刷完atv9 就变1+8了 哈哈.

**Comment by aylwin202406** [comment]
dear
能不能详细说明下，我的基础比较差，没搞明白。非常感谢
[cid:fd412c5a-9b80-45b8-b0b2-99268ca48b6d]
________________________________
发件人: ophub ***@***.***>
发送时间: 2024年6月11日 11:30
收件人: ophub/amlogic-s9xxx-armbian ***@***.***>
抄送: aylwin202406 ***@***.***>; Author ***@***.***>
主题: Re: [ophub/amlogic-s9xxx-armbian] O大，求助，我用的三陶教育s30盒子刷armbian mcu是s905d2，刷s905X固件，内存识别788M，不准确。 (Discussion #2363)


ophub/amlogic-s9xxx-openwrt#602<https://github.com/ophub/amlogic-s9xxx-openwrt/issues/602>

参考上面的方法，添加u-boot或者调整dtb

―
